share/man/man9/cdefs.9

```diff
@@ -0,0 +1,297 @@
+.\"-
+.\" Copyright (c) 2024 M. Warner Losh <imp@FreeBSD.org>
+.\"
+.\" SPDX-License-Identifier: BSD-2-Clause
+.\"
+.Dd July 3, 2024
```

Comment:
```suggestion
.Dd July 4, 2024
```

Reply:
yea, don't comment on the date. It's likely going to end up being July 14 or something, and I know to change it :)